src/contexts/app/AppStateContext.tsx

```
@@ -0,0 +1,228 @@
+    isSyncing: false,
+    lastSyncTime: safeGetItem(STORAGE_KEYS.LAST_SYNC_TIME, null),
+  };
+};
+
+const initialState: AppState = getInitialState();
+
+// State reducer
+function appStateReducer(state: AppState, action: AppStateAction): AppState {
+  switch (action.type) {
+    case 'SET_NAVIGATION_DATA':
+      const newNavigationData = { ...state.navigationData, ...action.payload };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)